Civi/RemoteTools/Api4/Api4.php

```diff
@@ -83,6 +83,16 @@ public function createGetAction(string $entityName): AbstractAction {
     return $this->createAction($entityName, 'get');
   }
 
+  /**
+   * @inheritDoc
+   */
+  public function deleteEntities(string $entityName, ConditionInterface $condition, array $options = []): Result {
+    return $this->execute($entityName, 'delete', [
+      'checkPermissions' => $options['checkPermissions'] ?? FALSE,
```

Comment:
*Not* checking permissions is the default?

Reply:
Yes, this is the case for the full `Api4Interface` as documented there.